src/audio/copier/copier.c

```diff
@@ -100,6 +101,41 @@ static struct comp_dev *copier_new(const struct comp_driver *drv,
 	}
 
 	dev->pipeline = ipc_pipe->pipeline;
+	/* Calculation of the period in which the component should be scheduled based on
+	 * the input buffer size.
+	 */
+	if (copier->gtw_cfg.dma_buffer_size > 0 && dev->pipeline->long_period) {
+		/* Size of the single sample in bytes. */
+		uint32_t sample_size = copier->base.audio_fmt.channels_count *
+				       (copier->base.audio_fmt.depth / 8);
```

Comment:
"/ 8" --> ">> 3"

Reply:
Its best to use the >> as not all compilers will do the right thing. e.g. could be optimal on xtensa, but on ARM it could divide using the same CC flags.